ts/backbone/views/Lightbox.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @prettier
+ */
+export const show = (element: HTMLElement): void => {
```

Comment:
As I mentioned above, it just feels wrong to write this code in typescript. :0/

Reply:
As discussed above, I’d like to write new code separately so we can more easily test it and know what to rip out later, e.g. `Signal.Backbone`. Plus, TypeScript still helps get the types and APIs right. Hope that’s cool 😄